src/js/base/module/Editor.js

```diff
@@ -733,13 +738,6 @@ define([
     this.empty = function () {
       context.invoke('code', dom.emptyPara);
     };
-
-    /**
-     * set height for editable
-     */
-    this.setHeight = function (height) {
-      $editable.outerHeight(height);
-    };
   };
```

Comment:
@lqez, @easylogic I thought below commit extracts setHeight as Editor API(eg. `summernote('editor.setHeight')`). Is there reason for providing setHeight as API?
https://github.com/summernote/summernote/commit/177dc4c4a5b8b401ae6d06b3def95fb0b31478ab